synthesizer/synthesizer.h

```diff
@@ -42,7 +42,7 @@ class Synthesizer {
       bool _active;
 
    protected:
-      float _sampleRate;
+      float _sampleRate{44100.};
```

Comment:
Yeah, I personally think `44100.0` is better because then you can easily spot the decimal point. A decimal point at the end (especially adjacent to a brace) is less recognizable.

Reply:
I agree. And so does the existing MuseScore codebase.

Reply:
changed to `44100.0f`